OptimizelySDK/Optimizely.cs

```diff
@@ -360,11 +365,14 @@ public Variation GetForcedVariation(string experimentKey, string userId)
             var experiment = Config.GetExperimentForVariationId(variation.Id);
 
             if (!string.IsNullOrEmpty(experiment.Key))
-                SendImpressionEvent(experiment, variation.Id, userId, userAttributes);
+                SendImpressionEvent(experiment, variation, userId, userAttributes);
```

Comment:
You are sending notification here and in line 373 as well. looks weird to me. please clear to me. If I am right, then why unit tests didn't get fail. We must cover, firenotification should be called how many times in case of activate, track or from where firenotification we are calling. 
@oakbani Just FYI.

Reply:
Nothing needs to worry about here. The type of Notifications here and in Line 373 are different. As we need to send two different type of notifications in this method. One for Decision and another for Feature Access.